packages/gui/package.json

```diff
@@ -106,6 +106,12 @@
       ]
     }
   },
+  "jest": {
+    "testPathIgnorePatterns": [
+      "/node_modules/",
+      "<rootDir>/tests/"
```

Comment:
Added because this is where the Playwright tests currently reside. We may want to move those somewhere else.